### DISCUSSION TITLE
Where do I put my dictionaries?

### DISCUSSION THREAD

**Discussion by michaelbeijer** [ilius/ayandict#1] [Q&A] [answered]
Perhaps a stupid question, but where do I put my dictionaries? I saw the ‘directory_list’ section in the config file but can't get it to work.

**Comment by ilius** [comment, marked as answer]
The default directory is `.stardict/dic` which is relative to your home directory. So you can create that folder and place your dictionaries in there. Or use another folder. Paths can be absolute/full or relative to you home.

**Reply by michaelbeijer** [reply]
Gotcha!

**Reply by ilius** [reply]
How is it going?

**Reply by michaelbeijer** [reply]
I got it working!

I generally use GoldenDict (this one: https://github.com/xiaoyifang/goldendict), but am always on the lookout for new terminology tools to test!
